code/datums/supplypacks.dm

```diff
@@ -1646,6 +1646,26 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 					)
 	containername = "shield control board crate"
 
+///////////// Bathroom Fixtures
+
+/datum/supply_packs/misc/toilet
+	name = "Urination Station Crate"
```

Comment:
Urination Station? Do we *have* to do the rhyming?

Reply:
I like this. It's super cringe worthy and should stay